tests/sentry/notifications/test_notifications.py

```diff
@@ -457,7 +461,7 @@ def test_sends_issue_notification(self, record_analytics):
         assert isinstance(msg.alternatives[0][0], str)
         assert "Hello world</pre>" in msg.alternatives[0][0]
 
-        attachment, text = get_attachment()
+        attachment, text = get_attachment(1)
```

Comment:
Do we know if the `response_num` needs to be passed in explicitly like this still? This was a different attempt to fix the tests.